### DISCUSSION TITLE
X-store 桃園也有

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS4719-C/Python-Community-GS4719-C#1111] [General]
X-store是7-11推出的無人商店（桃園大園區也有ㄌ）
我認爲無人商店的好處是可以直接的取得消費者喜好資料
了解客群取向與偏好
藉此分析每月或年營銷與銷售趨勢
或是藉由數據資料研擬新的銷售策略
https://www.bnext.com.tw/article/67024/7-11-x-store-5?

**Comment by wenchi9023** [comment]
無人商店最近好像有越來越多的趨勢，但感覺偷竊的問題也可能隨之增加? 畢竟監視器有死角之類的

話說，原po你的新聞連結和你提到的X-store好像無關，是不是有貼錯~

**Reply by wu51000** [reply]
對我貼錯惹qq謝謝你的提醒~我有重放了！有興趣的話可以再去看看～
